modules/golangci/lint.go

```diff
@@ -34,6 +45,10 @@ type LintRun struct {
 	Source *dagger.Directory
 	// +private
 	Path string
+	// +private
+	GoModCache *dagger.CacheVolume
+	// +private
+	GoBuildCache *dagger.CacheVolume
```

Comment:
:bulb: feature idea - it would be cool to have "derived" cache-volumes - the fact we need to pass multiple cache volumes around, when it's conceptually for the same thing is a bit weird.

Ideally you'd be able to pass one, and then do `CacheVolume.cacheVolume` or similar, and get another sub-cachevolume (either it would just modify the internal `name`, or it could even just mount a subpath of the cache volume into the container?).